cli/src/commands/add.rs

```diff
@@ -100,50 +197,55 @@ pub async fn add(
     }
 
     let existing_update_paths =
-        (update_paths(&mut conn, &payload.metadata.extension_name).await).unwrap_or_default();
+        (update_paths(&mut conn, &extension_name).await).unwrap_or_default();
 
     for upgrade_file in &payload.upgrade_files {
         let update_path = UpdatePath {
             source: upgrade_file.from_version.clone(),
             target: upgrade_file.to_version.clone(),
         };
         if !existing_update_paths.contains(&update_path) {
-            migration_content.push_str("SELECT pgtle.install_update_path('");
-            migration_content.push_str(&payload.metadata.extension_name);
+            migration_content.push_str("select pgtle.install_update_path('");
+            migration_content.push_str(&extension_name);
             migration_content.push_str("', '");
             migration_content.push_str(&upgrade_file.from_version);
             migration_content.push_str("', '");
             migration_content.push_str(&upgrade_file.to_version);
-            migration_content.push_str("', $SQL$");
+            migration_content.push_str("', $sql$");
```

Comment:
does this maybe need to be a uuid or other random string so malicious packages can't escape the quoting?

Reply:
Using a randomly generated suffix to prevent this attack.